### ISSUE TITLE
Docker fails on test with font license issue

### ISSUE THREAD

**Issue by boydkelly** [metanorma/metanorma#184]
I run the command:
````
podman run -v "$(pwd)":/metanorma/ -w /metanorma metanorma/metanorma metanorma compile -t iso -x xml test.adoc
````
And I get the output:

````
-----------------------------------------------------------------------
FONT LICENSE END ("Cambria")

Do you accept all presented font licenses, and want Fontist to download these fonts for you? => TYPE 'Yes' or 'No': [fontist] Aborting without proper fonts installed, make sure that you have set option --agree-to-terms
````
Other than I can't type YES even if I wanted to, in any case the Cambria font is a VERY poor default choice.  It does not support extended latin characters.
You can solve two problems in one shot here by substituting a widely available font that requires no Type YES To Accept.

google-noto-sans would be a safe choice.  No yes to accept garbage.  Great font with **world wide language support.**
There are many others you can choose.   Look at what the browsers support.   déja vu etc.

Cambria?????  Really????

**Comment by boydkelly** [comment]
ok I accepted with the switch.   But that still doesn't help me with displaying ɔ ɲ ɛ ŋ.   Github can do it with no problem.   Cambria can't.   And since I am on a slow expensive internet connection in west africa, I don't have the $$$ to download a non functional proprietary font every time I run the docker image.

**Comment by ronaldtse** [comment]
Appreciate raising the issue @boydkelly , this is an opportunity to explain a few nuances, please bear with me.

TL;DR: *Metanorma does not depend on proprietary fonts*.

Metanorma does not demand the use of any font or font style. By default, Metanorma uses Source Sans which uses an open license.

In any case -- are you writing an ISO deliverable? If you are, there is a separate discussion that can be made about ISO policy.

## Why is Metanorma asking me to install Cambria?

Short answer: because this is ISO policy.

1. Metanorma is a framework for creating standardization deliverables in standards organizations, such as ISO or ITU.
2. Standards organizations, such as ISO and ITU, mandate what content is acceptable from standards authors.
3. In this case, since the `iso` document class was selected, the generated document is meant to adhere to the rules of ISO.
4. ISO/CS (ISO Central Secretariat) requires all submitted documents to utilize Cambria as the default font.
5. Metanorma enables you to automatically install Cambria if you would want to generate such a document.


## What if I don't want Cambria?

Metanorma allows you to skip installing any font; you just need to enter "no" at the prompt. Then the outputs will utilize Source Sans which uses an open license.

In fact, there is a command that does not prompt the installation of fonts:

```sh
$ metanorma compile --no-install-fonts --continue-without-fonts foo.adoc
```

(see https://github.com/metanorma/metanorma-cli/blob/f286ff927b55781abddb855df0bbd60103890d50/lib/metanorma/cli/command.rb#L38-L41)

The caveat is: if you are writing an ISO standard, and don't use Cambria, you run the risk of ISO/CS rejecting your draft.

## How can I cache the font?

If the system already has the font (e.g. Cambria on Windows, or if you have MS Office installed on macOS), Metanorma's use of Fontist will detect and automatically used the installed font.

If you don't have the font, Metanorma prompts you to install it.

A number of proprietary fonts are free for download in certain conditions, often that:
* the font cannot be redistributed;
* the user must accept the font's EULA.

Metanorma, as a piece of software, cannot accept any license on behalf of the user, and for that reason Metanorma does not come distributed with any proprietary font (in all packaged forms).

This is a matter of standards body policy: some SDOs who are more committed to open source, such as ITU and OGC, only use openly licensed fonts.

There are two ways of caching proprietary fonts and using them with Metanorma:

1. If you use Docker, you can cache fonts using a docker volume to `/config/fonts` in the image:
```sh
$ docker run -v "$(pwd)":/metanorma/ \
  -v ${HOME}/.fontist/fonts/:/config/fonts \
  -w /metanorma metanorma/metanorma \
  metanorma compile -t iso -x xml test.adoc
```

If you agree with the terms of the font, you can do
```sh
$ docker run -v "$(pwd)":/metanorma/ \
  -v ${HOME}/.fontist/fonts/:/config/fonts \
  -w /metanorma metanorma/metanorma \
  metanorma compile -t iso -x xml --agree-to-terms test.adoc
```

2. If you use the single binary version (e.g. via Snap on Linux, Homebrew on macOS or Chocolatey on Windows), or the Ruby gem version, fonts are automatically cached at `~/.metanorma/fonts` so the download is a one-time process.


## Summary

Metanorma does not depend on proprietary fonts. The command shown indicates that the author is intending to generate an ISO deliverable, of which ISO demands the usage of Cambria. Metanorma allows skipping installation of Cambria anyway.

Hope this answers the question @boydkelly !
